drivers/net/ixgbe/ixgbe_ethdev.c

```diff
@@ -3984,8 +3984,6 @@ ixgbe_dev_link_update_share(struct rte_eth_dev *dev,
 	memset(&old, 0, sizeof(old));
 	rte_ixgbe_dev_atomic_read_link_status(dev, &old);
 
-	hw->mac.get_link_status = true;
-
```

Comment:
I think we can leave this one in for now (as there are a lot of bond driver changes in 17.11). This was added for a case where bond didn't come up with DPDK 17.02.
Rest looks ok.